Tests/UnitTests/Mocks/MockAttributionFetcher.swift

```diff
@@ -7,6 +7,8 @@
 
 class MockAttributionFetcher: AttributionFetcher {
 
+    var adServicesTokenCollectionCalled = false
```

Comment:
Nit: maybe move this right on top of `adServicesToken`? I think that's what other mocks do